src/components/ProjectDetails/ProjectDetails.js

```diff
@@ -43,6 +44,11 @@ const ProjectDetails = ( {
 
   const { writeLayoutToStorage } = useStoredLayout( "exploreObservationsLayout" );
 
+  const NONE = "NONE";
+  const JOIN = "JOIN";
+  const LEAVE = "LEAVE";
```

Comment:
These constants never change and don't need to be redefined on every render, so put them outside of the component.

Reply:
Done